core/primitives/src/types.rs

```diff
@@ -651,6 +651,35 @@ pub mod validator_stake {
                 stake_next_epoch: if is_next_epoch { self.stake() } else { 0 },
             }
         }
+
+        // TODO(chunk-validator-assignment) if `ValidatorStake` is recalculated every epoch, this should be a field of (new) `ValidatorStakeV2`?
+        // Pro: do the division only once. Con: Adding a field probably requires a `V2`, though is it worth it?
```

Comment:
Division is not that slow, I wouldn't be worried about it. We can always optimize if we find it to be an issue later.

Reply:
Addressed in 4c67030.